specification/columns/commitmentdiscountquantity.md

```diff
@@ -0,0 +1,54 @@
+# Commitment Discount Quantity
+
+Commitment Discount Quantity is the amount of a [*commitment discount*](#commitment-discount) purchased or accounted for in *commitment discount* related [*rows*](#glossary:row). The amounts are denominated in [*Commitment Discount Units*](#glossary:commitmentdiscountunit). The aggregated Commitment Discount Quantity across purchase records pertaining to a particular [Commitment Discount ID](#commitmentdiscountid) during its [*term*](#glossary:term) represents the total *Commitment Discount Units* acquired with that commitment discount. For committed usage, the Commitment Discount Quantity is either the number of *Commitment Discount Units* consumed by a [*row*](glossary:#row) that is covered by a *commitment discount* or is the unused portion of a *commitment discount* over a *charge period*. The CommitmentDiscountQuantity column only applies to *commitment discounts* and not [*negotiated discounts*](#glossary:negotiated-discount). Commitment Discount Quantity is commonly used in commitment discount analysis and optimization use cases.
+
+For usage-based *commitment discounts* (i.e. when *CommitmentDiscountCategory* is "Usage"), the Commitment Discount Quantity reflects the amount of predefined usage purchased or consumed. If *commitment flexibility* is applicable, this value may be further transformed based on additional, provider-specific requirements. For spend-based *commitment discounts*, the Commitment Discount Quantity reflects the prefined amount of spend purchased or consumed.
+
+Commitment Discount Quantity is commonly used in commitment discount analysis and optimization use cases. For purchases, the Commitment Discount Quantity helps customers track how many total *Commitment Discount Units* are available across *commitment discounts* at various points in time, and for committed usage, it helps customers track how efficiently their *commitment discounts* apply to their workloads. The *commitment discount's* utilization rate is used to measure this efficiency and can be derived by dividing the used portion of CommitmentDiscountQuantity by the sum of CommitmentDiscountQuantity for the *charge period*. With this information, customers can analyze their existing *commitment discount* strategy to make more informed decisions on additional purchases.
+
```

Comment:
Seems like in the process of adding/updating, we've duplicated a bunch of things? The last two lines or parts of these lines are repeated above.

Reply:
Not sure how that happened, will de-dupe this before tomorrow

Reply:
Resolved in subsequent commit